docs/contributing/Masterlist-Editing.html

```diff
@@ -10,13 +10,33 @@ <h3>General Hints</h3>
 <ul>
   <li>Look in the masterlists for examples, and see the <a href="https://loot-api.readthedocs.io/en/stable/metadata/introduction.html">Metadata Syntax documentation</a> for the details.</li>
   <li>Only make edits/additions you're confident about - otherwise create an issue in the masterlist repository's issue tracker, or discuss in the official LOOT forum thread.</li>
-  <li>If you're using Notepad++, you can set it to use spaces instead of tabs when you press the tab key on your keyboard, avoiding YAML parsing errors. To do this, go to Settings &gt; Preferences &gt; Tab Settings &gt; Replace by space. A value of 4 is most common.</li>
   <li>Make as many silent comments as you want within the masterlists - it doesn't hurt (within reason). However, make sure to put them <em>above</em> the line(s) that they refer to.</li>
   <li>Use blank lines to separate mods that are grouped together or where it helps readability.</li>
   <li>Where you put a plugin doesn't matter - you may want to see if there are any related plugins in the masterlist already and group them together, or you can just add the plugin to the end of the list.</li>
   <li>Test your changes before uploading them, to make sure you didn't make any syntax mistakes. See <a href="Quickly-Testing-Your-Masterlist-Changes.html">Quickly Testing Your Masterlist Changes</a> for instructions.</li>
 </ul>
 
+<h4>Notepad++</h4>
+<p>If you're using <a href="https://notepad-plus-plus.org/">Notepad++</a>, you can set it to use spaces instead of tabs when you press the tab key on your keyboard, avoiding YAML parsing errors. To do this, go to Settings &gt; Preferences &gt; Languages &gt; Tab Settings &gt; yaml &gt; Replace by space. A value of 4 is most common.</p>
```

Comment:
Since we use two-space indentation, we should probably recommend contributors use 2 spaces.

```suggestion
<p>If you're using <a href="https://notepad-plus-plus.org/">Notepad++</a>, you can set it to use spaces instead of tabs when you press the tab key on your keyboard, avoiding YAML parsing errors. To do this, go to Settings &gt; Preferences &gt; Languages &gt; Tab Settings &gt; yaml &gt; Replace by space. The masterlists use two-space indentation, so a tab size of 2 is recommended.</p>
```

Reply:
I assume you refer to the two spaces in front of `- name:`. While that would be true, all other tags that follow such as
```yaml
  - name:
    url:
    req:
    inc:
    tag:
    dirty:
    clean:
    msg:
```
are indented by 4 spaces. Since there should be more of those than `name`'s, wouldn't the `4` be more fitting?

Reply:
Sets under those keys would also be two-space indented as well, such as:

```yaml
  - name:
    msg:
      - type:
        content:
          - lang:
            text:
```

There's also this excerpt from the **Writing Style** section: "Child data nodes should be indented two spaces from their parents, for the best trade-off between compactness and readability." Though, yeah, keys on the same level are technically four-space indented. Maybe a good compromise would be something like "A tab size of 4 is most common. However, the masterlists use two-space indentation, so a tab size of 2 may be preferable."

Reply:
I'll go with that. Thanks.

Reply:
@pStyl3 indentation is measured relative to the previous level: if your example each list element is indented two spaces from the list's parent element (or document root) and each list element's keys are indented two spaces from that. I think @sibir-ine's suggestion is more correct, though I'd drop the bit about a tab size of 4 entirely to avoid confusion.

Reply:
Alright, will do!